### DISCUSSION TITLE
Want a free IntelliJ license ?

### DISCUSSION THREAD

**Discussion by maxandersen** [jbangdev/jbang#1155] [Q&A]
Hey,

I won the javaadvent raffle for a IntelliJ one year personal license for writing the 1. December article https://www.javaadvent.com/2021/12/jbang-gift-that-keeps-on-giving.html/amp

I don't need such a license thus thought I would give it away to anyone of the jbang contributors - meaning any one who contributed a bug report, docs, code, ideas etc. And been recognized on https://github.com/jbangdev/jbang/blob/main/CONTRIBUTORS.md

The first one from that list that responds to this with "I contribute to JBang and could use a IntelliJ license" gets it. 

The license is for one year for any of IntelliJ products. 

P.s. contributing something now for the first time also counts.

**Comment by fbricon** [comment]
I contribute to JBang but don't need an IntelliJ license :-)

**Reply by maxandersen** [reply]
So close ;)

**Comment by basil** [comment]
> P.s. contributing something now for the first time also counts.

I'd take it if #1158 is accepted.

**Reply by basil** [reply]
> Thanks for the contribution @basil!

You're welcome!

> if still interested in the intellij license go complete the ask on the discussion thread.

Completing the ask.

**Reply by maxandersen** [reply]
Close, but Read and follow the instructions Again please.

**Reply by maxandersen** [reply]
Especially this part:

"The first one from that list that responds to this with "I contribute to JBang and could use a IntelliJ license" gets it."

**Reply by basil** [reply]
Your wish is my command, Max! I contribute to JBang and could use a IntelliJ license.

**Reply by maxandersen** [reply]
You got it!

You don't seem to have email on GitHub profile or allow dm on your Twitter account. Can you let me know where to send it here or send me a dm on Twitter from the account listed on GitHub - then I can give me your license knowing it is the right person receiving it. Thanks.

**Reply by basil** [reply]
> You don't seem to have email on GitHub profile or allow dm on your Twitter account. Can you let me know where to send it here

```
curl https://api.github.com/repos/jenkinsci/jenkins/commits/685b60ca0b141a7fe578e54eec03e31db4006916 | jq '.commit.author'
```
